charter.md

```diff
@@ -53,15 +53,207 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+  - Call for removal?
+  - Reason for removal?
+  - Vote on removal?
+  - What are the public artifacts of the removal process?
+- [ ] What details of election results should be public/private?
+- [ ] Consequences of removal scenarios
+  - Are individuals ineligible for re-election?
+    - Should community members have an opinion in this decision?
+- [ ] When is removal appropriate?
+- [ ] How should CoC concerns be considered in the removal process? Should they be?
+- [ ] Guard against "popularity contests" or "burning at the person" in committee voting scenarios
+- [ ] Are there antitrust implications in the removal process?
+- [ ] Special elections may not have representative turnout
+  - Define guidance on acts and thresholds to consider in a vote for removal
+- [ ] Is removal an extraordinary power to grant?
+  - "Policies like this have to work for the good of the community both when used well and when used poorly."
+- [ ] Define removal process guidelines
+  - "brief rationale for removal" + short time + "brief rebuttal"
+  - Offer resignation as an option
+
+-->
+
+### No confidence
+
+In the event a Steering Committee member receives a vote of no confidence from
+two-thirds of the remaining Steering Committee members, that person must either
+resign or call a special election in which the project community will vote on
+whether that person remains or is removed from the Steering Committee. A
+majority of eligible voters participating in the election must vote in favor of
+the individual remaining; otherwise, they will be removed. The individual's
+participation in the Steering Committee will be suspended during the pendency
+of the special election.
+
+## Voting
+
+<!--
+
+COMMUNITY CONSIDERATIONS
+
+Voting
+
+- [ ] Better define voting requirements in specific scenarios
+  - Consider 5/7, as 4/6 is a simple majority of the entire committee
+  - Clarify requirements respective to size of committee and round up e.g., 7 * 2/3 = 4.6666..., which would be 5 votes out of 7
+  - Are there scenarios where unanimity should be considered?
+  - [Stephen] What about a combination of both?
+
+-->
+
+In the course of the committee's operations, members will be expected to vote
+on decisions within the body's purview.
+
+These votes may proceed on agreed-upon platforms by the committee, such as:
+
+* a pull request
+* an issue
+* a Steering Committee [meeting](#meetings)
+* a mailing list
+
+Unless otherwise specified by a process, the requirement for passing a vote is
+a **_two-thirds supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* 5 members in attendance would be required to vote during a meeting
+
+### Abstention
+
+For any self-elected reason, members of the committee may decide to abstain
+from a vote.
+
+Abstaining members will only be considered as contributing to quorum, in the
```

Comment:
Abstaining is effectively a no, since you need 5 votes to pass anything on a 7 member committee -- as you've set it up the denominator never changes; usually abstaining changes the denominator in my understanding.

I believe quorum hasn't been defined above in this document yet.

Reply:
oh, it's defined below, I would move that section up.